custom-elements.json

```diff
@@ -9423,15 +9423,6 @@
               "name": "selectedTab",
               "privacy": "private"
             },
-            {
```

Comment:
I hadn't planned to strip private members from the manifest until we had a reason to expose the manifest publicly. 

But changes to private members shouldn't cause a diff and unnecessarily increase review times. I'll follow up with a PR that removes private members.